docs/Modders/Mod-file-structure/2-Civilization-related-JSON-files.md

```diff
@@ -91,6 +92,37 @@ Each nation has the following structure:
       These notes are __only__ valid when playing on generated maps, loaded maps from map editor get no "regions" and startBias is processed differently (but you can expect single-entry startBias to work best).
 [^V]: See [Supply Leader Voices](../Images-and-Audio.md#supply-leader-voices)
 
+## Personalities.json
+
+This file contains all Personalities for computer players.
+
+Each nation has the following structure:
+
+| Attribute | Type | Default | Notes |
+| --------- | ---- | ------- | ----- |
+| name | String | Required | |
+| preferredVictoryType | String | Neutral | The victory type major civilizations will pursue (need not be specified in [VictoryTypes.json](5-Miscellaneous-JSON-files.md#victorytypesjson)) |
+| [`<stats>`](3-Map-related-JSON-files.md#general-stat) | Float | 5 | Amount of focus on the stat the computer player will have. Typically ranges from 0 (no focus) to 10 (maximum focus) |
+| military | Float | 5 | Amount of focus on the military growth the computer player will have. Typically ranges from 0 (no focus) to 10 (maximum focus) |
+<!-- | warMongering | Float | 5 | Amount of focus on the declaring war the computer player will have. Typically ranges from 0 (no focus) to 10 (maximum focus) |-->
```

Comment:
"-1" just is a negative focus, 11 is I guess slightly higher than double focus. Edited it to say "double" focus instead of "maximum" focus for the sake of mad modders (unless we decide to enforce a hard limit. Actually, maybe the hard limit should be done via uniques)